packages/archway-proto/scripts/aminos.js

```diff
@@ -0,0 +1,31 @@
+export const AMINO_MAP = {
+  // PUT YOUR AMINO names here...
```

Comment:
Do you think we should also add mappings for the Rewards module here?

Reply:
I did not enable aminoEncoding as I thought we only needed rpc. If you want me to enable that would this be the correct syntax in the aminos.js file?
```
'/archway.rewards.v1beta1.MsgCreateValidator': {
    aminoType: 'cosmos-sdk/MsgCreateValidator' // I'm assuming this remains same?
  },
```